model_analyzer/result/run_config_result.py

```diff
@@ -23,33 +23,33 @@
 
 
 @total_ordering
-class ModelResult:
+class RunConfigResult:
     """
-    A class that represents the result of
-    a single run. This ModelResult belongs
+    A class that represents the group of measurements from
+    a single RunConfig. This RunConfigResult belongs
     to a particular ResultTable
     """
 
-    def __init__(self, model_name, model_config, comparator, constraints=None):
+    def __init__(self, model_name, model_configs, comparator, constraints=None):
         """
         Parameters
         ----------
         model_name: str
             The name of the model
-        model_config : ModelConfig
-            The model config corresponding to this run
-        comparator : ResultComparator
+        model_configs : list of ModelConfigs
+            The list of model configs corresponding to this RunConfig
+        comparator : RunConfigResultComparator
             An object whose compare function receives two
-            results and returns 1 if the first is better than
+            RunConfigResults and returns 1 if the first is better than
             the second, 0 if they are equal and -1
             otherwise
         constraints: dict
-            keys are metrics and values are 
-            constraint_type:constraint_value pairs
+            Used to determine if a RunConfigResult passes or fails
```

Comment:
do we have support for per-model constraints? Will a simple dict cover that?

Reply:
No, we don't yet. Not sure if this is already filed (if not I will file it today. Measurement changes (just) added support for multi-model weightings.

Reply:
TMA-571 was filed to track this